server/neptune/workflows/internal/activities/github.go

```diff
@@ -204,3 +204,40 @@ func (a *githubActivities) FetchRoot(ctx context.Context, request FetchRootReque
 		LocalRoot: localRoot,
 	}, nil
 }
+
+type CompareCommitRequest struct {
+	DeployRequestRevision  string
+	LatestDeployedRevision string
+	Repo                   internal.Repo
+}
+
+type CompareCommitResponse struct {
+	DeployRequestRevisionAheadBy int
```

Comment:
Whats the point of returning the quantity here? Isn't just knowing whether it's ahead, behind or diverged enough?

If thats the case, maybe something like a typed string/enum would be better.
See: https://github.com/lyft/deployapi/blob/76de5a28d996c3805ab4753def02f4aae7a82051/pkg/github/github.go#L31-L38